lib/reqTemplate.js

```diff
@@ -263,11 +264,11 @@ function replaceComplexTemplates(part, subSpec, globals) {
             return replaceComplexTemplates(part, elem, globals);
         });
     } else if (subSpec && subSpec.constructor === String || subSpec === '') {
-        if (/\{.*\}/.test(subSpec)) {
+        if (/\{[^\}]+\}/.test(subSpec)) {
             // There is a template, now we need to check it for special stuff we replace
-            if (/^\{[^\}]+\}$/.test(subSpec)) {
+            if (/^\{\{[^\{\}]+\}\}$/.test(subSpec)) {
```

Comment:
We should also support single expressions with single braces. It's not that useful, but we need it for consistency.